app/src/main/java/edu/calvin/cs262/cs262d/eventconnect/views/AddEvent.java

```diff
@@ -53,35 +104,85 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
+    /**
+     *
+     * @param view
+     */
     public void onCreateEventClicked(View view) {
         //access the data from the UI elements
         //TODO: Enforce a title and description??
         String title = eventTitle.getText().toString();
         String desc = eventDescription.getText().toString();
-        String host = eventHost.getText().toString();
         String loc = eventLocation.getText().toString();
-        String date = eventDate.getText().toString();
+        //String date = eventDate.getText().toString();
```

Comment:
I don't know why this is commented out but if it's not needed, it should be deleted.